lib/formatOrder.js

```diff
@@ -1,7 +1,50 @@
 var sorting = require('postcss-sorting')
 
 function formatOrder (root, params) {
-  var sortOrder = params.stylelint['declaration-block-properties-order']
+  var sortOrder =
+    processOrderPluginRule(params.stylelint['order/declaration-block-property-groups-structure']) ||
+    processStylelintRule(params.stylelint['declaration-block-properties-order'])
+
+  if (!Array.isArray(sortOrder)) {
+    return
+  }
+
+  var sort = sorting({
+    'order': [
+      'at-rules',
+      'custom-properties',
+      'dollar-variables',
```

Comment:
I think this should be the first, because currently dollar variables goes after return.